tests/routes/reviews.test.js

```diff
@@ -88,9 +86,7 @@ describe('TEST Suites for the REVIEW ACTIONS', () => {
         request(server)
           .post('/api/v1/businesses/1/reviews')
           .set({ authorization: token })
-          .send({
-            review: 'The food at your restaurant was heavenly',
-          })
+          .send(reviewSeed.setInput('The food at your restaurant was heavenly'))
```

Comment:
Same thing here, I don't really see any reason for using `.setInput`

Reply:
Should I remove it?